Makefile

```diff
@@ -77,7 +77,8 @@ ALL_SCRIPTS     := $(shell find . -name "*.sh")
 
 PINNED_DEPENDENCIES := \
 	github.com/apache/thrift@v0.0.0-20161221203622-b2a4d4ae21c7 \
-	github.com/go-sql-driver/mysql@v1.5.0
+	github.com/go-sql-driver/mysql@v1.5.0 \
+	go.opentelemetry.io/otel/metric@v0.27.0 # There are breaking changes in v0.28.0
```

Comment:
we will have to adjust code to that, but that can be in follow up PR.

Reply:
Yeah, that's my plan as well. 👍